src/main/java/com/massivecraft/factions/config/file/MainConfig.java

```diff
@@ -1165,7 +1165,7 @@ public boolean isBankFactionPaysLandCosts() {
         private int height = 17;
         private int width = 49;
         private boolean showFactionKey = true;
-        private boolean showNeutralFactionsOnMap = true;
+        private boolean showNeutralFactions = true;
```

Comment:
If this is going to change, might want to change it in the transition config too ;)